### PR TITLE
Backport 15809, 15802, 15792, 15913

### PR DESCRIPTION
**Pull request by PastaPastaPasta**

### CODE REVIEW

File: src/dummywallet.cpp [hunk 1]
```diff
@@ -33,17 +33,48 @@ class DummyWalletInit : public WalletInitInterface {
...
+        "-zapwallettxes=<mode>",
+        "-discardfee=<amt>",
+        "-fallbackfee=<amt>",
+        "-mintxfee=<amt>",
+        "-paytxfee=<amt>",
+        "-txconfirmtarget=<n>",
+        "-hdseed=<hex>",
+        "-mnemonic=<text>",
+        "-mnemonicpassphrase=<text>",
+        "-usehd", "-keepass",
```
Comment (UdjinM6) [inline src/dummywallet.cpp:60]: 15913: split
```suggestion
        "-usehd",
        "-keepass",
```

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

**Comment by UdjinM6** [comment]
ping?

**Comment by PastaPastaPasta** [comment]
Done, thanks for the ping :)

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**